src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -202,7 +202,7 @@ public function testParseUnquotedScalarStartingWithReservedIndicator($indicator)
 
     public function getReservedIndicators()
     {
-        return array(array('@'), array('`'));
+        return array(array('@'), array('`'), array('|'), array('>'));
```

Comment:
Are they really reserved indicators actually ? AFAIK, they have a meaning. They are not just reserved